### PR TITLE
ENH : GH11729 added index parameter in series to_string

### LINKED ISSUE #11729: Add option to hide index in pandas.Series.to_string
In the Dataframes.to_string function there is a option to not print the index:

index : bool, optional

    whether to print index (row) labels, default True

Can this be added to the Series function to_string?

http://pandas.pydata.org/pandas-docs/version/0.17.0/generated/pandas.Series.to_string.html

### PR DESCRIPTION
**Pull request by varunentropy**
Fixes #11729 
Please Review .

### CODE REVIEW

File: pandas/core/format.py [hunk 1]
```diff
@@ -126,14 +126,15 @@ def to_string(self):
 class SeriesFormatter(object):
 
     def __init__(self, series, buf=None, length=True, header=True,
-                 na_rep='NaN', name=False, float_format=None, dtype=True,
-                 max_rows=None):
+                 na_rep='NaN', index=True, name=False, float_format=None,
```
Comment (jreback) [inline pandas/core/format.py]: are these in the same order as the ``DataFrameFormatter`` ?

File: pandas/core/format.py [hunk 2]
```diff
@@ -241,7 +242,10 @@ def to_string(self):
             fmt_values.insert(row_num + n_header_rows, dot_str)
             fmt_index.insert(row_num + 1, '')
 
-        result = self.adj.adjoin(3, *[fmt_index[1:], fmt_values])
+        if self.index:
+            result = self.adj.adjoin(3, *[fmt_index[1:], fmt_values])
+        else:
```
Comment (jreback) [inline pandas/core/format.py]: is this really what dataframe does ? (e.g. with the adjoin)? for the no index case

File: pandas/tests/test_format.py [hunk 3]
```diff
@@ -3181,6 +3181,15 @@ def test_to_string_float_na_spacing(self):
                     '4       NaN')
         self.assertEqual(result, expected)
 
+    def test_to_string_without_index(self):
+        s= Series([1, 2, 3, 4])
```
Comment (jreback) [inline pandas/tests/test_format.py]: add the issue number as a comment here

**Comment by jreback** [comment]
pls add a whatsnew note (enhancements is ok)

**Comment by jreback** [comment]
@varun-kr can you update

**Comment by varun-kr** [comment]
@jreback  Updated code . Please review .

It seems that there is some problem with the build . 
ERROR: test_get_all_data (pandas.io.tests.test_data.TestYahooOptions) is failing for many builds.

**Comment by jreback** [comment]
just push again; the option tests occasionally fail

**Comment by varun-kr** [comment]
@jreback  Done.

File: pandas/core/format.py (hunk 2 above)
Comment (jreback) [inline pandas/core/format.py]: any thoughts here?

Comment (varun-kr) [inline pandas/core/format.py]: Dataframe does not use adjoin. In dataframe we use a [list](https://github.com/pydata/pandas/blob/784445cbe81cc8b2da4e2324689f175cb2a2edbf/pandas/core/format.py#L492) and insert index into that. I was expecting that dataframe and series formatter code will be similar but this is not the case .

File: pandas/tests/test_format.py [hunk 4]
```diff
@@ -3181,6 +3181,16 @@ def test_to_string_float_na_spacing(self):
                     '4       NaN')
         self.assertEqual(result, expected)
 
+    def test_to_string_without_index(self):
+        #GH 11729 Test index=False option
+        s= Series([1, 2, 3, 4])
+        result = s.to_string(index=False)
+        expected = (u(' 1\n') +
+                    ' 2\n' +
```
Comment (jreback) [inline pandas/tests/test_format.py]: @jorisvandenbossche does this look right? (e.g. I know we add a space, but seems odd with no index....)

Comment (jorisvandenbossche) [inline pandas/tests/test_format.py]: It seems more logical to me that the space is not there if no index is added.

But, `DataFrame.to_string` also does this:

```
In [13]: print df.to_string()
   a  b  c
0  1  b  1
1  1  b  2
2  2  b  3
3  2  a  4

In [14]: print df.to_string(index=False)
 a  b  c
 1  b  1
 1  b  2
 2  b  3
 2  a  4
```

Of course not necessarily a reason to do it here as well

**Comment by jreback** [comment]
@varun-kr thanks!

can you open an issue to re-examine whether passing ``index=False`` should add those leading spaces (I think not).

**Pull request merged**